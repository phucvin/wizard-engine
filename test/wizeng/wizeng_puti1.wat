(module
  (import "wizeng" "puti" (func $puti (param i32)))
  (func $main (export "main")
    (call $puti (i32.const 44112233))
    (call $puti (i32.const -99667788))
  )
)
