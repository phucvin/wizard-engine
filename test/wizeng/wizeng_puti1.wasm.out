44112233-99667788